### DISCUSSION TITLE
Irritations/Questions

### DISCUSSION THREAD

**Discussion by ohquait** [wwmm/easyeffects#3227] [General]
**First i have to say THANK YOU!
Users on PCLiunxOS are very happy with pulseeffects and easyeffects.**

I'm just a packager, not a coding-guru.

Irritation:
in the wiki
https://github.com/wwmm/easyeffects/blob/master/COMMUNITY_PRESETS_GUIDELINES.md
it says from version 7.2.0.......
but the current release is 7.1.7,
do the instructions apply from 7.0.2 and it's just a swap of the numbers?

Note/question for distributions:
pulseeffects also works great under pipewire (I use it too),
but easyeffects can only work under pipewire?

Can someone please confirm this for me?

Before I package the community presets as an rpm, I would like to know more.
That will be helpful for users.

best regards

**Comment by violetmage** [comment]
Yes, it should say 7.1.7 for the release of community presets.

Also yes, easyeffects only works with pipewire.

**Reply by wwmm** [reply]
It was supposed to be 7.2 because of the amount of changes but I forgot about this when making the release.

**Reply by ohquait** [reply]
**Thanks !**
